app/src/main/java/com/example/teamyuml/decentworkmobile/database/DBHelper.java

```diff
@@ -0,0 +1,144 @@
+package com.example.teamyuml.decentworkmobile.database;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.SQLException;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteException;
+import android.database.sqlite.SQLiteOpenHelper;
+import com.example.teamyuml.decentworkmobile.model.*;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+/**
+ * Class is responsible for creating copy of the database from asset file to
+ * memory of device
+ */
+public class DBHelper extends SQLiteOpenHelper {
+    private static String DB_PATH = "/data/data/com.example.teamyuml.decentworkmobile/databases/";
+    private static String DB_NAME = "DecentWorkMobile.sqlite";
+    private SQLiteDatabase myDataBase;
+    private final Context myContext;
+
+    /**
+     * Constructor
+     * Takes and keeps a reference of the passed context in order to access to the application assets and resources.
+     * @param context
+     */
+    public DBHelper(Context context) {
+        super(context, DB_NAME, null, 1);
+        this.myContext = context;
+    }
+
+    /**
+     * Creates a empty database on the system and rewrites it with your own database.
+     * */
+    public void createDataBase() throws IOException{
+
+        boolean dbExist = checkDataBase();
+
+        if(dbExist){
```

Comment:
1. `if () {`
2.  Use `if (!ckeckDataBase()) {` Then you don't have to write `// do nothing...`